v1/index.js

```diff
@@ -22,7 +22,6 @@ module.exports = {
   BinaryHTTPEmitter,
   StructuredHTTPReceiver,
   BinaryHTTPReceiver,
-  Cloudevent: event,
```

Comment:
Breaking change...

Reply:
Yes. This is a PR with a breaking change. We'll cut a new version.